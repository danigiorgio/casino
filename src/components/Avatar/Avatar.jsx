import PropTypes from 'prop-types';

const Avatar = ({ src, alt }) => {
  return <img className="ui avatar image" src={src} alt={alt} />;
};

Avatar.propTypes = {
  src: PropTypes.string,
  alt: PropTypes.string,
};

export default Avatar;
