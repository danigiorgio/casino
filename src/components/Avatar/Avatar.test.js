import { render } from '@testing-library/react';

import Avatar from './Avatar';

describe('<Avatar />', () => {
  it('should render Avatar', () => {
    const { container } = render(<Avatar />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
