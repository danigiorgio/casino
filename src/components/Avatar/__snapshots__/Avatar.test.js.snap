// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Avatar /> should render Avatar 1`] = `
<img
  class="ui avatar image"
/>
`;
